addons/website/views/website_form_templates.xml

```diff
@@ -1,140 +0,0 @@
-                                    <div class="row s_col_no_resize s_col_no_bgcolor">
-                                        <label class="col-form-label col-sm-auto s_website_form_label" style="width: 200px" for="contact6">
-                                            <span class="s_website_form_label_content">Your Question</span>
-                                        </label>
-                                        <div class="col-sm">
-                                            <textarea id="contact6" class="form-control s_website_form_input" name="Description" required=""></textarea>
-                                        </div>
-                                    </div>
-                                </div>
-                                <div class="form-group col-12 s_website_form_field s_website_form_dnone">
-                                    <div class="row s_col_no_resize s_col_no_bgcolor">
-                                        <label class="col-form-label col-sm-auto s_website_form_label" style="width: 200px" for="contact7">
-                                            <span class="s_website_form_label_content">Email To</span>
-                                        </label>
-                                        <div class="col-sm">
-                                            <input id="contact7" type="hidden" class="form-control s_website_form_input" name="email_to"/>
-                                        </div>
-                                    </div>
-                                </div>
-                                <div class="form-group col-12 s_website_form_submit" data-name="Submit Button">
-                                    <div style="width: 200px;" class="s_website_form_label"/>
-                                    <a href="#" role="button" class="btn btn-primary btn-lg s_website_form_send">Submit</a>
-                                    <span id="s_website_form_result"></span>
-                                </div>
-                            </div>
-                        </form>
-                    </div>
-                </section>
-            </div>
-        </xpath>
-    </template>
-
-    <record id="contactus_thanks" model="website.page">
-        <field name="name">Thanks (Contact us)</field>
-        <field name="type">qweb</field>
-        <field name="url">/contactus-thank-you</field>
-        <field name="website_indexed" eval="False"/>
-        <field name="is_published">True</field>
-        <field name="key">website.contactus_thanks</field>
-        <field name="arch" type="xml">
-            <t name="Thanks (Contact us)" t-name="website.contactus_thanks">
-                <t t-call="website.layout">
-                  <div id="wrap">
-                    <div class="oe_structure" id="oe_structure_website_form_contact_us_thanks_1"/>
```

Comment:
Why were those oe_structure IDs element removed? The templates are moved to website but those are simply gone.
Is you blamed those line, you see it is there on purpose:
Added everywhere with (among other commits): https://github.com/odoo/odoo/commit/4f15c939a78b4f1f20377d62e89157dfe628d910
In order to achieve this: https://github.com/odoo/odoo/commit/dd139948f0049b3676b859495ba9b2d43436f9c1

Was it discussed with someone or just a mistake to remove those?

Reply:
I think it was qsm meant in my other contactus PR ? What do you think ?

> I don't think there is a reason for the "thank you" page to not be fully static too, confirm that with @sbuhl. But in that case, there won't be two oe_structure around the page content and the customization of those in current user database would have to be migrated.

>  Le but c'est que ces pages "contactus", "thank you" etc soient exactement comme des pages que l'utilisateur aurait créées lui-meme.

If I drag and drop snippets to create a page, there only will be an `oe_structure` on the `#wrap div`

Reply:
Ok so it was discussed and on purpose :+1: 
Not sure why there was a PR change and those review/comments are lost but fine

Reply:
> I don't think there is a reason for the "thank you" page to not be fully static too

Really sad to open a ticket and don't have an reference on the thank you page :/